docs/src/content/docs/pt-br/guides/components.mdx

```diff
@@ -60,23 +59,76 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 Você pode mostrar uma interface com abas utilizando os componentes `<Tabs>` e `<TabItem>`.
 Cada `<TabItem>` tem que ter uma `label` para mostrar aos usuários.
+Use o atributo opcional `icon` para incluir um dos [ícones predefinidos do Starlight](#todos-os-ícones) ao lado da label.
 
 ```mdx
 # src/content/docs/exemplo.mdx
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <Tabs>
-	<TabItem label="Estrelas">Sirius, Vega, Betelgeuse</TabItem>
-	<TabItem label="Luas">Io, Europa, Ganymede</TabItem>
+	<TabItem label="Estrelas" icon="star">
+		Sirius, Vega, Betelgeuse
+	</TabItem>
+	<TabItem label="Luas" icon="moon">
+		Io, Europa, Ganymede
```

Comment:
Should "Ganymede" be "Ganímedes"? ([reference](https://pt.wikipedia.org/wiki/Gan%C3%ADmedes_(sat%C3%A9lite))) If yes, the example below should also be updated.